api/nodes/serializers.py

```diff
@@ -218,7 +219,7 @@ class NodeSerializer(TaxonomizableSerializerMixin, JSONAPISerializer):
                                             'level project by submitting the appropriate fields in the request body, '
                                             'and some information will not change. By default, the description will '
                                             'be cleared and the project will be made private.')
-
+    templated_by_count = ser.SerializerMethodField(help_text='How many projects have used this as a template.')
```

Comment:
This diverges from how we do counts for forks and links. Shouldn't this be a relationship?

Reply:
@sloria The requirements for the page don't call for an actual templated_by relationship. We could certainly add one, but I don't know if that's a worthwhile addition if we don't want to be able to get the nodes. It probably should go in the node's meta field if it isn't, rather than attributes, however.

Reply:
Fine by me.